illumio_connector.py

```diff
@@ -375,9 +375,7 @@ def _handle_create_service_binding(self, param):
         workload_hrefs_list = self.handle_comma_seperated_string(
             param["workload_hrefs"]
         )
-        virtual_service_href = illumio.convert_draft_href_to_active(
-            param["virtual_service_href"]
-        )
+        virtual_service_href = param["virtual_service_href"]
```

Comment:
Was this meant to be reverted?

Reply:
As SDK expects active HREF and if not provided, SDK returns a valid message. So, adding a draft HREF is a valid negative test case and should fail.

Reply:
So is the expectation that playbooks will update the HREF in pre/post-processing? I think we should aim to accommodate both here and just catch the case where a draft HREF is passed that doesn't have an active version.

Reply:
I disagree with this, but we can run it by Rei.

Reply:
Based on the discussion today, let's switch this back to convert to active. For the case where a user tries this action with an unprovisioned virtual service, we can either catch the resulting error from the service binding create, or do a `get_by_reference` call on the active HREF to check that it exists (this may be a simpler solution due to the nature of the service bindings POST).

Reply:
Made required changes.

Reply:
Thanks - how are we handling the unprovisioned virtual service case?

Reply:
Updated the code which will catch the exception raised for unprovisioned virtual service and display appropriate message